client/components/common/tce-core/ElementDiscussion.vue

```diff
@@ -112,6 +112,14 @@ export default {
       this.editorBus.emit(events.RESOLVE, { id, contentElementId, resolvedAt });
     }
   },
+  watch: {
+    isVisible(val) {
+      const { elementId } = this.$route.query;
```

Comment:
Ideally shouldn't be hooked to the router. ContentElement (parent) has handlers to figure out if it is selected.